shell-ui/src/library.js

```diff
@@ -0,0 +1,19 @@
+import {
+  AUTHENTICATED_EVENT,
+  LANGUAGE_CHANGED_EVENT,
+  THEME_CHANGED_EVENT,
+} from './events';
+import { version } from '../package.json';
+
+window.shellUI = {
+  ///spread shellUI to keep all versions libraries
+  ...window.shellUI,
```

Comment:
Are we expecting multiple "shellUI"s in a single window?

Reply:
Ideally we don't want this but this could be the case in the futur : for example a micro app component could import shell-ui `1.0.0` and on the same page another component could rely on `1.1.0` ending up in the necessity to keep both libraries